components/domActions.js

```diff
@@ -13,12 +14,21 @@ export const cancelTodo = () => {
   textArea.value = "";
 };
 
+export function emptyListMessage() {
+  if (todoWrapper.innerHTML === "") {
```

Comment:
A more conclusive check might be to check the length of `todos` array.
For this project we can be sure that todoWrapper will be empty if no todo is added but in general we can't say that about DOM. So it's better check the source of data(which todos for us)

```suggestion
  if (todos.length === 0) {
```